docs/reference/modules/network.asciidoc

```diff
@@ -119,9 +123,10 @@ sorted by IPv4/IPv6 stack preference, then by reachability. If you set a
 address for node-to-node communication, you should explicitly set
 `network.publish_host`.
 
-Both of the above settings can be configured just like `network.host` -- they
-accept IP addresses, host names, and
-<<network-interface-values,special values>>.
+This setting accepts IP addresses, host names, and
+<<network-interface-values,special values>>, but does not accept arrays.
```

Comment:
> but does not accept arrays 

This setting does in fact accept a list of values, much like how it accepts a DNS name that resolves to multiple addresses. It's probably not very useful to do that, and I'd rather not spell this out explicitly in the documentation because I feel it will cause more confusion than it resolves.